packages/fether-react/src/Send/TxForm/TxDetails/TxDetails.js

```diff
@@ -9,18 +9,29 @@ import { fromWei, toWei } from '@parity/api/lib/util/wei';
 
 class TxDetails extends Component {
   renderDetails = () => {
-    const { estimatedTxFee } = this.props;
+    const { estimatedTxFee, token, values } = this.props;
 
-    return estimatedTxFee
-      ? `${this.renderCalculation()}
+    if (
+      !estimatedTxFee ||
+      !values.gasPrice ||
+      !values.amount ||
+      !token.address
+    ) {
+      return `Missing input fields...`;
```

Comment:
Can we keep the line break so that the message is cantered?
```suggestion
      return `
Missing input fields...`;
```

Reply:
Yes, I need to be more careful. I've added a comment for future reference